src/applications/pensions/config/form.js

```diff
@@ -706,7 +706,7 @@ const formConfig = {
           },
         },
         marriageHistory: {
-          title: (form, { pagePerItemIndex }) =>
+          title: (form, { pagePerItemIndex } = { pagePerItemIndex: 0 }) =>
```

Comment:
I don't think this is a pattern supported by the Forms Library. This is probably just effectively passing 0 in both cases, but the title should be updated by the `uiSchema` configuration anyway, so this might not matter.